src/procedures/ModifyShareholderData.ts

```diff
@@ -17,12 +17,23 @@ import { Shareholder, SecurityToken } from '../entities';
 
 const { dateToBigNumber } = conversionUtils;
 
+/**
+ * Procedure that modifies data of a list of investors
+ */
 export class ModifyShareholderData extends Procedure<
   ModifyShareholderDataProcedureArgs,
   Shareholder[]
 > {
   public type = ProcedureType.ModifyShareholderData;
 
+  /**
+   * - Update a list of investors data
+   *
+   * Note that this procedure will fail if you're trying to revoke a shareholder's KYC
```

Comment:
Remove redundancy

... you're trying to set the dates to 0 (there is a special "RevokeKyc" procedure for that)